docs/tools/generate.fsx

```diff
@@ -1,11 +1,20 @@
-//#load "../../.paket/load/netstandard2.0/docs/docs.group.fsx"
-#load "../../.paket/load/netstandard2.0/docs/FSharp.Compiler.Service.fsx"
 #load "../../.paket/load/netstandard2.0/docs/FSharp.Literate.fsx"
 #load "../../.paket/load/netstandard2.0/docs/Fable.React.fsx"
+
 #if !FAKE
 #r "netstandard"
 #endif
 
+// HACK: force usage of Fsharp.Compiler.Services
+// or the indirect reference from FSharp.Literate will fail to load
+let dummy (pos: FSharp.Compiler.Range.pos) =
+    pos.Column
+FSharp.Compiler.Range.mkPos 1 1 |> dummy
+
+// HACK: Force usage of Fable.Core
+// or the indirect reference from Fable.React will fail to load
+typeof<Fable.Core.EraseAttribute>
+
```

Comment:
Just wow!  It really works but looks sad...
I think that we should report this as `dotnet fsi` issue to `dotnet/fsharp` repo.
What do you think @thinkbeforecoding ?

Reply:
Yes, and I was even not ale to make the one for FSharp.Compiler work with a typeof<...> I don't really know why...

Reply:
I'd really like to have @dsyme or @cartermp take on the issue...

Reply:
I'm not aware of the issues around doc generation using dotnet fsi. Odd problem

Reply:
@dsyme  FCS and Fable.Core are inderect references for FSharp.Literate and Fable.React. Just adding a #r to the assembly at the begining is not enough to make it work, dotnet fsi complains when loading FSharp.Literate or Fable.React that indirect dependencies cannot be found.
The hack is to actively use code from FCS and Fable.Core in dummy expression to force the assembly to load. Then the rest of the code works as expected.

Reply:
https://github.com/dotnet/fsharp/pull/7725

If this still doesn't work, can you point me to a repro and I will try to figure out what is needed.

Thanks

KEvin